pandas/io/common.py

```diff
@@ -74,7 +74,8 @@ def is_url(url) -> bool:
 def _expand_user(
     filepath_or_buffer: FilePathOrBuffer[AnyStr],
 ) -> FilePathOrBuffer[AnyStr]:
-    """Return the argument with an initial component of ~ or ~user
+    """
+    Return the argument with an initial component of ~ or ~user
        replaced by that user's home directory.
```

Comment:
this introduces a weird alignment here

Reply:
this PR is just D213, the alignment will be fixed with D207/D208

Reply:
so some overlap here with #31890, and D207/D208 will need to be rerun after D209/D213 merged

Reply:
i'll push the fix in this PR. (prob thur)

Reply:
done